SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Ingestion/Models/StartServiceLog.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+
```

Comment:
The other concrete log classes have an attribute here for the json identifier - should it be here, too? It's used for type identification when deserializing derived types